.gitignore

```diff
@@ -15,6 +15,7 @@ pids
 libs/
 node_modules/
 bower_components/
+*.js.map
```

Comment:
I think its fine to leave in there, just to make sure those don't get committed. But IDE settings should be such that it doesnt produce them. Turn off the watchers in Webstorm: Preferences -> Languages & Frameworks -> Typescript -> Generate Source maps.